src/components/VCalendar/util/timestamp.ts

```diff
@@ -0,0 +1,296 @@
+  while (--days >= 0) nextDay(timestamp)
+  return timestamp
+}
+
+export function prevDay (timestamp: VTimestamp): VTimestamp {
+  timestamp.day--
+  timestamp.weekday = ((timestamp.weekday || 0) + 6) % DAYS_IN_WEEK
+  if (timestamp.day < DAY_MIN) {
+    timestamp.month--
+    if (timestamp.month < MONTH_MIN) {
+      timestamp.year--
+      timestamp.month = MONTH_MAX
+    }
+    timestamp.day = daysInMonth(timestamp.year, timestamp.month)
+  }
+  return timestamp
+}
+
+export function getWeekdaySkips (weekdays: number[]): number[] {
+  let skips: number[] = [1, 1, 1, 1, 1, 1, 1]
+  let filled: number[] = [0, 0, 0, 0, 0, 0, 0]
+  for (let i = 0; i < weekdays.length; i++) {
+    filled[weekdays[i]] = 1
+  }
+  for (let k = 0; k < DAYS_IN_WEEK; k++) {
+    let skip = 1
+    for (let j = 1; j < DAYS_IN_WEEK; j++) {
+      let next = (k + j) % DAYS_IN_WEEK
+      if (filled[next]) {
+        break
+      }
+      skip++
+    }
+    skips[k] = filled[k] * skip
+  }
+  return skips
+}
+
+export function createDayList (start: VTimestamp, end: VTimestamp, now: VTimestamp,
+  weekdaySkips: number[], max: number = 42): VTimestamp[] {
+  let current = copyTimestamp(start)
+  let stop = getDayIdentifier(end)
+  let currentIdentifier = 0
+  let days: VTimestamp[] = []
+
+  while (currentIdentifier !== stop && days.length < max) {
+    currentIdentifier = getDayIdentifier(current)
+    if (weekdaySkips[current.weekday] === 0) {
+      current = nextDay(current)
+      continue
+    }
+    let day = copyTimestamp(current)
+    updateFormatted(day)
+    updateRelative(day, now)
+    days.push(day)
+    current = nextDays(current, weekdaySkips[current.weekday])
+  }
+
+  return days
+}
+
+export function createIntervalList (timestamp: VTimestamp, first: number,
+  minutes: number, count: number, now?: VTimestamp): VTimestamp[] {
+  let intervals: VTimestamp[] = []
+
+  for (let i = 0; i < count; i++) {
+    let mins: number = (first + i) * minutes
+    let int: VTimestamp = copyTimestamp(timestamp)
+    int.hasTime = true
+    int.hour = Math.floor(mins / MINUTES_IN_HOUR)
+    int.minute = mins % MINUTES_IN_HOUR
+    int.time = getTime(int)
+    if (now) {
+      updateRelative(int, now, true)
+    }
+    intervals.push(int)
+  }
+
+  return intervals
+}
+
+export function createNativeLocaleFormatter (locale: string, getOptions: VTimestampFormatter<object>): VTimestampFormatter<string> {
+  let emptyFormatter: VTimestampFormatter<string> = (t, s) => ''
+
+  if (typeof Intl === 'undefined' || typeof Intl.DateTimeFormat === 'undefined') {
+    return emptyFormatter
+  }
+
+  return (timestamp, short) => {
+    try {
+      const intlFormatter = new Intl.DateTimeFormat(locale || undefined, getOptions(timestamp, short))
+      let time = `${padNumber(timestamp.hour, 2)}:${padNumber(timestamp.minute, 2)}`
+      let date = timestamp.date
+      return intlFormatter.format(new Date(`${date}T${time}:00+00:00`))
+    } catch (e) {
+      window.console.log(e)
```

Comment:
remove dev code

Reply:
Most of this code was copied from Date Picker code, looks like the Intl.DateTimeFormat class likes to throw errors... so I don't think so :-/